option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message FriendDekaronRequest
{
 required U64Type player_id = 1;
}
